src/Swift.Bindings/tests/IntegrationTests/CryptoKit/CryptoKit.cs

```diff
@@ -38,7 +38,7 @@ public Nonce(Data data)
                 _payload = NativeMemory.Alloc(PayloadSize);
                 SwiftIndirectResult swiftIndirectResult = new SwiftIndirectResult(_payload);
 
-                void* metadata = Test.Runtime.GetMetadata(data);
+                void* metadata = Runtime.GetMetadata(data);
```

Comment:
The signature should (eventually) be TypeMetadata instead of void*

Reply:
Replaced with `TypeMetadata`. We will automate generation of bindings when more support is available.